learn/react-native/custom-icon-fonts.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Custom Icon Fonts"
+id: "custom-icon-font"
+sidebar_label: "Custom Icon Fonts"
+---
+---
+
+Custom Icon font families can now be imported into the studio using this feature. It can be imported in two ways.
+
+- Using the File Explorer
+- Using the Select Icon dialog
+
+## Using File Explorer
+
+1. Go to File Explorer and click Import Resource to import a new icon family.
+
+![](/learn/assets/file_explorer_import_resource.png)
```

Comment:
Error: Image static/learn/assets/file_explorer_import_resource.png used in ../learn/react-native/custom-icon-fonts.md not found.